config/job_conf.xml.sample_advanced

```diff
@@ -762,6 +762,13 @@
                 will be terminated by Galaxy.
          -->
         <limit type="walltime">24:00:00</limit>
+	<!-- total_walltime:
+		Total walltime that jobs may not exceed during during a
+		set period.  If total walltime of finished jobs exceeds
+		this value, any new jobs are paused.  window is a number
```

Comment:
window is referring to an attribute here, so it should probably be lowercase.  Maybe wrap in quotations marks?